src/Foundation/NSUrlSessionHandler.cs

```diff
@@ -159,12 +173,45 @@ public NSUrlSessionHandler (NSUrlSessionConfiguration configuration)
 			inflightRequests = new Dictionary<NSUrlSessionTask, InflightData> ();
 		}
 
+#if !MONOMAC && !WATCH
+
+		void AddNotification ()
+		{
+			if (!isBackgroundSession && notificationToken == null)
+				using (var notification = new NSString ("UIApplicationWillResignActiveNotification"))
```

Comment:
same comments as earlier:

* why not use `UIApplication.WillResignActiveNotification` instead of your own constant ? Minimally it avoid the memory allocation and any chance of a typo. Also if it ever gets obsoleted we'll not get a compiler warning to update this code.

* why not follow the advice on the field ? `"Use UIApplication.Notifications.ObserveWillResignActive helper method instead."` ?